File: C/Day_15.c
// Get a three-digit nmber from user and print the sum of digits

#include <stdio.h>
int main(){
    int num, digit, digit_1, digit_2, digit_3;
    printf("Enter a three-digit number: ");
    scanf("%d", &num);
    digit_1 = num/100;
    digit_3 = num%10;
    digit = num%100;
    digit_2 = digit/10;
    printf("Sum of digits: %d", digit_1+digit_2+digit_3);
    return 0;
}

File: Python/Day_15.py
# Get a three-digit number from user and print the sum of the digits

num = int(input("Enter a three-digit number: "))
digit_1 = num//100
digit_3 = num%10
digit = num%100
digit_2 = digit//10
print("Sum of digits:", digit_1+digit_2+digit_3)
